it/uk/gov/hmrc/agentrelationships/connectors/DesConnectorSpec.scala

```diff
@@ -45,6 +59,11 @@ class DesConnectorSpec extends UnitSpec with OneAppPerSuite with WireMockSupport
       givenDesReturnsServerError()
       an[Exception] should be thrownBy await(desConnector.getNinoFor(mtdItId))
     }
+
+    "record metrics for GetRegistrationBusinessDetailsByMtdbsa" in {
```

Comment:
This test is dependent on the other tests being run, right? i.e. it will fail if run by itself?
If so it would seem better if it was independent, i.e. would pass when run by itself. I'm pretty sure there are examples of tests like that in our other repos.